contract/src/main/kotlin/com/r3/corda/sdk/token/contracts/states/AbstractToken.kt

```diff
@@ -26,4 +26,7 @@ abstract class AbstractToken : ContractState {
 
     /** For creating a copy of an existing [AbstractToken] with a new holder. */
     abstract fun withNewHolder(newHolder: AbstractParty): AbstractToken
+
+    /** Get issuer for the token. */
+    abstract val issuer: AbstractParty // TODO Will issuer always be a well known identity?
```

Comment:
It's because i am lazy, and i just wanted to call token.issuer, instead of differentiating between amount.token.issuer for fungible and token.issuer for non-fungible